week1/k8s-resources/job-ml.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: batch/v1
+xapiVersion: batch/v1
```

Comment:
hm, not sure here is such thing as `xapiVersion`

Reply:
my bad

Reply:
Fixed